R/filter_confirmation.R

```diff
@@ -0,0 +1,233 @@
+#' Filter Confirmed Assessments
+#'
+#' The function filters observation using a condition taking subsequent
+#' observations into account. For example, it could select all observations with
```

Comment:
What happens in case where a patient has no records meeting the criteria - i assume no records are returned? could be worth explaining this somewhere in this paragraph. you'd then know the patient couldn't have confirmed BOR of CR or PR and you'd move on to checking if they're SD, etc

Reply:
If the criteria are not met, no records are returned.

I think it is to much detail for the description section.

I have added a details sections to clarify. Apologies for missing it before.

Reply:
sounds good thanks - will take a look later when you commit